extension-androidauto/CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog for Mapbox Maps Android Auto Extension SDK
+
+Mapbox welcomes participation and contributions from everyone.
+
+# main
+## Features ✨ and improvements 🏁
+## Bug fixes 🐞
+
+# extension-androidauto-v0.1.0-beta.1 April 5, 2022
+
+## Features ✨ and improvements 🏁
+* Add `MapboxCarMap`. Constructing this object will add mapbox map to the head unit. ([#1004](https://github.com/mapbox/mapbox-maps-android/pull/1004))
+* Add `MapboxCarMapObserver` and `MapboxCarMapSurface` which allows you to build customizations for the `MapboxMap`. ([#1004](https://github.com/mapbox/mapbox-maps-android/pull/1004))
+* Add `MapboxCarMapGestureHandler` and `DefaultMapboxCarMapGestureHandler` for handling panning and zooming gestures. (#[1159](https://github.com/mapbox/mapbox-maps-android/pull/1159))
+* Add an example to the `:android-auto-app:` module, it includes a 3D location puck, and follows the car location. ([#1004](https://github.com/mapbox/mapbox-maps-android/pull/1004))
+* Show Mapbox Logo and a Compass in an example with the `CarMapWidgets`. The `BitmapWidget` is available starting with Maps [v10.4.0](https://github.com/mapbox/mapbox-maps-android/releases/tag/android-v10.4.0). ([#1004](https://github.com/mapbox/mapbox-maps-android/pull/1004))
+
+## Dependencies
+* Android Auto car library `androidx.car.app:app:1.1.0`
+* Supports any version of Mapbox Maps 10.+
```

Comment:
I think we couldn't use widgets with Mapbox Maps below v10.4.0. Did you try what will happen if these are v10.0.0 is used with Android Auto extension?

Reply:
Updated the READMEs and docs in https://github.com/mapbox/mapbox-maps-android/pull/1267

Reply:
It's the `android-auto-app` that breaks compatibility with `10.0`. The `extension-android` is compatible with 10.+

The compatibility checklist is perfect thanks! After #1267 merges this comment stays as is right? It could reference the README compatibility checklist